Sources/Widgets/Representations/RectangleContextRepresentation/index.js

```diff
@@ -59,53 +59,35 @@ function vtkRectangleContextRepresentation(publicAPI, model) {
     const dataset = vtkPolyData.newInstance();
 
     if (state.getVisible()) {
-      const bounds = state.getBounds();
-      const point1 = [bounds[0], bounds[2], bounds[4]];
-      const point2 = [bounds[1], bounds[3], bounds[5]];
+      const normal = state.getDirection();
+      const vertical = state.getUp();
+      const horizontal = state.getRight();
```

Comment:
it is confusing to get a variable to recompute it just after.

Reply:
That's a leftover from before using Right and Up, we don't even need the direction/normal anymore... will remove, thanks.